daskernetes/core.py

```diff
@@ -228,10 +229,23 @@ def scale_up(self, n, **kwargs):
         """
         pods = self.pods()
 
-        out = [
-            self.core_api.create_namespaced_pod(self.namespace, self.pod_template)
-            for _ in range(n - len(pods))
-        ]
+        for i in range(3):
+            try:
+                out = [
+                    self.core_api.create_namespaced_pod(self.namespace, self.pod_template)
+                    for _ in range(n - len(pods))
+                ]
+                break
+            except client.rest.ApiException as e:
+                if 'ServerTimeout' in str(e):
```

Comment:
There's probably a better way to check for this one, let me look.

Reply:
I think checking for e.status = 500 and e.reason = 'ServerTimeout' is a better check here, but otherwise this looks good.